src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts

```diff
@@ -151,10 +151,11 @@ export class CellToggleMoreActions {
 
 	public onInit(elementRef: HTMLElement, context: CellContext) {
 		this._moreActionsElement = elementRef;
+		this._moreActionsElement.setAttribute('aria-haspopup', 'menu');
 		if (this._moreActionsElement.childNodes.length > 0) {
 			this._moreActionsElement.removeChild(this._moreActionsElement.childNodes[0]);
 		}
-		this._moreActions = new ActionBar(this._moreActionsElement, { orientation: ActionsOrientation.VERTICAL });
+		this._moreActions = new ActionBar(this._moreActionsElement, { orientation: ActionsOrientation.VERTICAL, ariaLabel: localize('moreActionsLabel', "More") });
```

Comment:
Just use a single localized constant for this and the one on line 347 - no reason to duplicate strings